vector/src/main/res/layout/merge_composer_layout.xml

```diff
@@ -48,7 +48,7 @@
         tools:text="@tools:sample/first_names"
         tools:visibility="gone" />
 
-    <TextView
+    <im.vector.riotx.core.platform.EllipsizingTextView
```

Comment:
Can you describe the use case when we should use EllipsizingTextView please? (Maybe in contributing.md)

Reply:
If you add it, it's avoid this issue: https://issuetracker.google.com/issues/121092510

Reply:
ok, thanks